src/care/array_utils.h

```diff
@@ -1177,29 +1244,32 @@ int ArrayMinMax(care::host_device_ptr<const globalID> arr, care::host_device_ptr
  *             Otherwise, returns 1.
  *             care::local_ptr API to support calls from within RAJA contexts.
  * ************************************************************************/
-template <typename T, typename ReducerType>
-inline int ArrayMinMax(care::local_ptr<const T> arr, care::local_ptr<int const> mask, int n, double *outMin, double *outMax) {
+template <typename T>
+CARE_HOST_DEVICE inline int ArrayMinMax(care::local_ptr<const T> arr, care::local_ptr<int const> mask, int n, double *outMin, double *outMax) {
    bool result = false;
-   ReducerType min, max;
+   // a previous implementation had min and max as a templated type and then used std::numeric_limits<T>::lowest() and 
```

Comment:
Add an issue for this.

Reply:
I made github tickets. Previously I got confused and made jira tickets instead